packages/shared/src/libs/helpers/change-case/change-case.helper.ts

```diff
@@ -1,10 +1,14 @@
 import { caseTypeToFunction } from "./libs/maps/maps.js";
 import { type CaseType } from "./libs/types/types.js";
 
-const changeCase = (input: string, caseType: CaseType): string => {
+const changeCase = (
+	input: Array<string> | string,
```

Comment:
```suggestion
	input: string,
```
you need to do it before passing as an argument to the function